bindings/rust/bench/src/harness.rs

```diff
@@ -143,6 +145,22 @@ macro_rules! test_tls_bench_harnesses {
                     }
                 }
             }
+
+            #[test]
+            fn test_transfer() {
+                // use a large buffer to test across TLS record boundaries
+                let mut buf = [0x56u8; 1000000];
+                let (mut harness, mut crypto_config);
+                for cipher_suite in [AES_128_GCM_SHA256, AES_256_GCM_SHA384].iter() {
```

Comment:
Why does this test need to iterate through all these cipher suite/EC choices?

Reply:
Because cipher suite/EC choices affect the way data gets encrypted/written into records and then transferred, I thought it'd be good to test all of them. Something like certificate signature type or client auth (not in this PR but in progress) I think wouldn't need to be tested then.

Reply:
I'm just not sure what is being tested here. If you're only testing the happy path of the round_trip_transfer function, you don't need to iterate through all of these options.

Reply:
The throughput benchmarks test different cipher suites but not different EC groups, and I want to test everything that the benchmarks use. I'll take out the ec_group then because the benchmarks don't bench it, but keep testing cipher suites.